medor_custom/data/data.xml

```diff
@@ -37,5 +37,32 @@
             <field name="client_id" ref="data_oauth_provider_client_medor_coop"/>
         </record>
 
+        <record id="data_medor_dummy_trial_journal" model="account.journal">
```

Comment:
why it doesn't go in demo.xml ?

Reply:
Because it is not demo data: it is a fake journal and fake product to make a free trial subscription.

Reply:
well if it's not for test. putting fake as name of a used product and journal is not a good idea from an accounting point of view. I mean that the administration can be picky on this. 

Why not free_trial  instead of dummy_trial ? It has the advantage to be self explained.

Reply:
free_trial is better 👍